packages/flutter/lib/src/widgets/navigation_toolbar.dart

```diff
@@ -0,0 +1,136 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math' as math;
+
+import 'package:flutter/rendering.dart';
+
+import 'basic.dart';
+import 'framework.dart';
+
+/// [NavigationToolbar] is an unthemed layout helper to position 3 groups
```

Comment:
Do we really need to say "unthemed" here? I don't think we qualify any widgets this way.

This widget doesn't position 3 groups of widgets, it positions three widgets.